tests/test_normalized.cpp

```diff
@@ -0,0 +1,221 @@
+/*****************************************************************************
+ *   Copyright (C) 2004-2013 The PaGMO development team,                     *
+ *   Advanced Concepts Team (ACT), European Space Agency (ESA)               *
+ *   http://apps.sourceforge.net/mediawiki/pagmo                             *
+ *   http://apps.sourceforge.net/mediawiki/pagmo/index.php?title=Developers  *
+ *   http://apps.sourceforge.net/mediawiki/pagmo/index.php?title=Credits     *
+ *   act@esa.int                                                             *
+ *                                                                           *
+ *   This program is free software; you can redistribute it and/or modify    *
+ *   it under the terms of the GNU General Public License as published by    *
+ *   the Free Software Foundation; either version 2 of the License, or       *
+ *   (at your option) any later version.                                     *
+ *                                                                           *
+ *   This program is distributed in the hope that it will be useful,         *
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of          *
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the           *
+ *   GNU General Public License for more details.                            *
+ *                                                                           *
+ *   You should have received a copy of the GNU General Public License       *
+ *   along with this program; if not, write to the                           *
+ *   Free Software Foundation, Inc.,                                         *
+ *   59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.               *
+ *****************************************************************************/
+
+// Test code for the normalized meta-problem
+
+#include <iostream>
+#include <algorithm>
+#include <cmath>
+#include <vector>
+#include <cassert>
+#include "../src/pagmo.h"
+#include "../src/rng.h"
+#include <boost/random/uniform_real.hpp>
+
+using namespace pagmo;
+
+const double EPS = 10e-9;
+
+#define PRINT_VEC(x) do {                             \
```

Comment:
I don't like that we redefine this PRINT_VEC macro everywhere, it's already here:

https://github.com/esa/pagmo/blob/master/tests/test_decompose.cpp#L41
https://github.com/esa/pagmo/blob/master/tests/test_shifted.cpp#L38

I would either drop it and let people debug the failing unit tests themselves, or define it elsewhere in a header file, perhaps dedicated for debugging/pretty printing, (e.g. some "debug_utils.h"). I would also find such file quite convenient for development.